src/Components/Catalog/Film/Film.js

```diff
@@ -1,16 +1,17 @@
+import { Link } from 'react-router-dom';
 import style from '../../../styles/Film.module.css'
 
 
-const Film = ({film,i}) => {
+const Film = ({film}) => {
     return ( 
-        <li className={style.poster} key = {i}>
-                <img className={style.image} src = {film.poster}/> 
-                <p className={style.title}>{film.title}</p>
-                <p className={style.year}>{film.year}</p>
-                <button className={style.info}>Info</button>
-                <button className={style.remove}>Remove</button>
-            </li>
+        <li className={style.poster}>
+            <img className={style.image} src = {film.poster} alt = "Poster"/> 
+            <p className={style.title}>{film.title}</p>
+            <p className={style.year}>{film.year}</p>
+            <button className={style.info}><Link to= {`/FilmPage/${film.id}`}>Info</Link></button>
+            <button className={style.remove}><Link to='#'>Remove</Link></button>
```

Comment:
try to add styles for link without wrapping link in button

Reply:
done